tasks/poll-index.js

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
this should not go into tasks as it is part of the app, right?

Reply:
But it should be a separate node process running along in the same instance right?

Reply:
noooo, this runs in the browser not in node

Reply:
or do you want to run it in the API?

Reply:
we could also run this in the API and on an update trigger an apollo subscription

Reply:
no, I think in the browser it makes more sense & it is more simple